src/components/hubspotForm.ts

```diff
@@ -0,0 +1,109 @@
+import { FC, createElement as h, useState, MouseEvent } from "react";
+import { TextField, Button, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles(theme => ({
+  title: {
+    marginTop: theme.spacing(3),
+    marginBottom: 0,
+  },
+  textField: {
+    marginRight: theme.spacing(1),
+  },
+  flexGrow: {
+    flexGrow: 2,
+  },
+  submitButton: {
+    marginTop: theme.spacing(1.5),
+  },
+  flex: {
+    display: "flex",
+  },
+  signedUp: {
+    color: theme.palette.secondary.main,
+  },
+}));
+
+interface Props {}
+
+const HubspotForm: FC<Props> = ({}) => {
+  const classes = useStyles();
+  const [name, setName] = useState<string>("");
+  const [email, setEmail] = useState<string>("");
+  const [submitted, setSubmitted] = useState<boolean>(false);
+
+  const handleSubmit = (e: MouseEvent) => {
+    e.preventDefault();
+    var xhr = new XMLHttpRequest();
+    var url = `https://api.hsforms.com/submissions/v3/integration/submit/${process.env.HUBSPOT_ACCOUNT_NUMBER}/${process.env.HUBSPOT_API_KEY}`;
+    var data = {
+      fields: [
+        {
+          name: "email",
+          value: email,
+        },
+        {
+          name: "firstname",
+          value: name,
+        },
+      ],
+      context: {
+        pageUri: "https://blog.clausehound.com/",
```

Comment:
do we want to show which specific page they are submitting from? or is this enough information?

Reply:
actually a good idea to include the specific pages